src/Type/Enum/PostObjectFieldFilterEnumType.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
This file name needs to change to: PostObjectFieldFormatEnumType

Reply:
Not sure how unit tests didn't bomb on this 🤔

Reply:
!!! ... is composer saving the day here?